style.css

```diff
@@ -1,635 +1,1223 @@
+/* @import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap"); */
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;600;800&display=swap");
+
 * {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+  background: #e5e5e5;
   margin: 0;
   padding: 0;
-  text-decoration: none;
-  font-family: 'Poppins', sans-serif;
-  overflow-x: hidden;
+  font-weight: 400;
+  font-size: 16px;
+  color: black;
+  overflow-x: clip !important;
 }
 
-body {
-  background-color: rgb(248, 248, 248);
+.overflowHidden {
+  overflow: hidden;
+  margin: 0;
+  touch-action: none;
+  -ms-touch-action: none;
+  height: 100vh;
+}
+
+.mobile-menu {
   width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  right: 0;
+  position: absolute;
+  background: #fff;
+  mix-blend-mode: multiply;
+  transform: translate(-100%);
+  transition: all 1s ease-in-out;
+  padding: 1.5rem;
+}
+
+.show {
+  transform: translate(0);
+  backdrop-filter: blur(8px);
+  background: #6070ff;
+  width: 100%;
+  height: 100vh;
+}
+
+ul,
+li {
+  list-style: none;
+  padding-left: 0;
 }
 
 header {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  width: 100%;
   display: flex;
-  align-items: center;
   justify-content: space-between;
-  padding: 15px 20px;
-  max-width: 100%;
+  align-items: center;
+
+  /* padding: 20px 36px 15px 24px; */
+  box-sizing: border-box;
+  background-color: #fff;
+  z-index: 1;
 }
 
-li {
-  cursor: pointer;
+.mobile-menu-inner ul {
+  display: flex;
+  flex-direction: column;
+  margin-top: 1rem;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+.mobile-nav li {
+  margin-bottom: 1.5rem;
+}
+
+.cancel {
+  display: block;
+  width: 1.5rem;
+  padding: 0 1rem;
+  color: #fff;
+  font-size: 2rem;
+  margin-left: auto;
+}
+
+i {
+  font-style: normal;
+  font-size: large;
 }
 
-.logo a {
-  font-size: 1.6rem;
-  white-space: nowrap;
-  font-weight: 800;
+a {
+  font-family: "Poppins", sans-serif;
+  font-weight: 700;
+  font-size: 18px;
+  text-decoration: none;
   color: #6070ff;
 }
 
-.profile img {
-  object-fit: cover;
-  width: 100%;
-  height: 100%;
-  border: 10px;
-  border-radius: 15px;
+.nav-mobile a {
+  color: #fff;
+  font-size: 2rem;
+  font-weight: 600;
+  overflow: hidden;
 }
 
-.menu ul a img {
-  width: 40px;
-  height: 40px;
+.menu-bar {
+  display: none;
 }
 
-.main-section {
-  background: url(image-source/header-shapes\ mobile@2x.svg);
-  background-repeat: no-repeat;
-  background-position: center;
-  background-size: cover;
-  height: 80vh;
+.modal {
+  position: fixed;
   width: 100%;
+  height: fit-content;
```

Comment:
-[Optional] kindly `height: fit-content;` is using  the available space, so please delete it to have better ui 👍👌

Reply:
Thanks for approving @binhussen